graylog2-server/src/main/java/org/graylog2/dashboards/Dashboard.java

```diff
@@ -26,6 +26,10 @@
 
     void setTitle(String title);
 
+    Map<String, Map<String, Integer>> getPositions();
+
+    void setPostions(Map<String, Map<String, Object>> positions);
```

Comment:
- Typo: `setPostions` -> `setPositions`
- Instead of `Map<String, Map<String, Object>>` we should use a structured & stricter typed parameter type here, e.g. an auto-value based value class.